test/e2e/chaos/networkchaos/network_peers_crossover.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package networkchaos
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+	"github.com/chaos-mesh/chaos-mesh/test/e2e/util"
+)
+
+// This test case is for https://github.com/chaos-mesh/chaos-mesh/issues/1450
+// For example, if the source is A, B, and the target is C, D, and the direction is both,
+// now the connection between A and B will also be affected by this chaos, this is unexpected.
+func TestcasePeersCrossover(
+	ns string,
+	cli client.Client,
+	networkPeers []*corev1.Pod,
+	ports []uint16,
+	c http.Client,
+) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	By("prepare experiment playground")
+	for index := range networkPeers {
+		err := util.WaitE2EHelperReady(c, ports[index])
+
+		framework.ExpectNoError(err, "wait e2e helper ready error")
+	}
+
+	result := probeNetworkCondition(c, networkPeers, ports, false)
+	framework.ExpectEqual(len(result[networkConditionBlocked]), 0)
+	framework.ExpectEqual(len(result[networkConditionSlow]), 0)
+
+	var (
+		testDelayTcParam = v1alpha1.TcParameter{
+			Delay: &v1alpha1.DelaySpec{
+				Latency:     "200ms",
+				Correlation: "25",
+				Jitter:      "0ms",
+			},
+		}
+		testDelayDuration      = pointer.StringPtr("9m")
+		testDelaySchedulerSpec = &v1alpha1.SchedulerSpec{
+			Cron: "@every 10m",
+		}
```

Comment:
seems that we can remove the scheduler config

Reply:
fixed